KinoPub/components/VideoDescriptionPanel.brs

```diff
@@ -43,14 +43,22 @@ end sub
 
 sub itemReceived()
     print "VideoDescriptionPanel:itemReceived"
+    
+    filesCount = m.readItemTask.content.item.videos[0].files.Count()
+    selectedFile = 1
+    
+   	if filesCount = 1
+		selectedFile = 0 
+	end if
+
     if false
```

Comment:
You can just remove this part. I think this was here when I was experimenting with Video.